modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/ObjectEntryModelListener.java

```diff
@@ -117,6 +134,50 @@ private void _executeObjectActions(
 		}
 	}
 
+	private AuditMessage _getAuditMessage(
+		String eventType, ObjectEntry objectEntry) {
+
+		AuditMessage auditMessage = AuditMessageBuilder.buildAuditMessage(
+			eventType, objectEntry.getModelClassName(),
+			objectEntry.getObjectEntryId(), null);
+
+		JSONObject additionalInfoJSONObject = auditMessage.getAdditionalInfo();
+
+		Map<String, Serializable> values = objectEntry.getValues();
+
+		for (Map.Entry<String, Serializable> entry : values.entrySet()) {
+			additionalInfoJSONObject.put(entry.getKey(), entry.getValue());
+		}
+
+		return auditMessage;
+	}
+
+	private List<Attribute> _getModifiedAttributes(
+		ObjectEntry originalObjectEntry, ObjectEntry objectEntry) {
+
+		List<Attribute> attributes = new ArrayList<>();
+
+		Map<String, Serializable> originalValues =
+			originalObjectEntry.getValues();
+		Map<String, Serializable> values = objectEntry.getValues();
+
+		for (Map.Entry<String, Serializable> entry :
```

Comment:
@rodrigopaulino I would do
`_getModifiedAttributes(Map<String, Serializable> originalValues,  objectEntry.getValues())` just to buy two lines 😆 

But there's no need to change this.

Reply:
Oh right! Nice catch. I'll wait for the results (which will most certainly fail) and change this. Thx, @carolmariaabb

Reply:
Done